lighthouserc.cjs

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    ci: {
+      upload: {
+        target: 'temporary-public-storage',
+      },
+      collect: {
+        staticDistDir: './dist',
+        "url": [
+          "http://localhost/"
+        ]
+      },
+      asserts: {
+        assertions: {
+          'categories:accessibility': ['warn', { minScore: 1 }],
```

Comment:
nit: Can we make the syntax similar, e.g. either use `1` or `1.0` ?

Reply:
TBH I think aiming for 90% or better is probably a healthier metric for the MVP phase. Usually 100% becomes an impossible standard